CHANGELOG.md

```diff
@@ -214,7 +214,7 @@ After:
 
 ## Under the hood
 
-- Declare compatibility with dbt v0.21.0, which has no breaking changes for this package ([#398](https://github.com/fishtown-analytics/dbt-utils/pull/398))
+- Declare compatibility with dbt v0.21.0, which has no breaking changes for this package ([#398](https://github.com/https://github.com/dbt-labs/dbt-utils-analytics/dbt-utils/pull/398))
```

Comment:
This file's URLs look pretty busted (two `github.com` references, `dbt-utils-analytics`) - did a find and replace go awry? 

Beside that, I'm not averse to old changelog entries using the old format - I wouldn't have brought it up if the switch had been clean, but since it's not I wanted to point out that reverting the file was also a valid option - these links were accurate at the time they were built, and are basically read-only, and will continue to resolve in the right place.

Reply:
Thank you for noticing that find-and-replace gone awry! 👀 💪 

Will make this update and re-request your review.

Agreed that the old URLs should continue to resolve just fine. I'd still prefer to update all references of "github.com/fishtown-analytics" to "github.com/dbt-labs/dbt-utils" so that it's clear during any `grep` operation that the repo is up-to-date and fully reflects the org name change.